app/models/exchange_rates/period_list.rb

```diff
@@ -0,0 +1,15 @@
+require 'api_entity'
+
+class ExchangeRates::PeriodList
+  include ApiEntity
+
+  attr_accessor :year
+
+  has_many :exchange_rate_years, class_name: 'ExchangeRates::Year'
+  has_many :exchange_rate_periods, class_name: 'ExchangeRates::Period'
+
+  def self.find(year, opts = {})
```

Comment:
You could just use `.all(year: 2022)` and get rid of this find method which feels weird for a list anyway :)

Reply:
I have discussed this with Will and we are going to stick with the .find override because the year params need to be optional on the frontend views and backend API.